### ISSUE TITLE
You Absolute Legend`

### ISSUE THREAD

**Issue by MushroomFleet** [kijai/ComfyUI-FluxTrainer#1]
First, Thanks for all your hard work and efforts! You are a Legend :)

This error is caused by the final line, `you have sentencepiece 0.2.0 which is incompatible. image-reward 1.5 requires timm==0.6.13, but you have timm 1.0.8 which is incompatible.`

ArgosTranslate was unable to install due to the previous installation of a newer Dependancy.

error copy:
`aeiou 0.0.20 requires soundfile<=0.10.2, but you have soundfile 0.12.1 which is incompatible. argostranslate 1.9.3 requires sentencepiece==0.1.99, but you have sentencepiece 0.2.0 which is incompatible. image-reward 1.5 requires timm==0.6.13, but you have timm 1.0.8 which is incompatible.`

From the argostranslate github, it seems that the 0.2.0 was a fix for newer python, regardless we can change the sentence piece deps to `0.1.99` and then it will require the correct argotranslate 1.9.3
https://github.com/argosopentech/argos-translate/issues/405

> I reduced the requirements for Sentencepiece, to match argostranslate 1.9.3
`sentencepiece>=0.1.99`

TLDR, Sentencepiece required by ArgosTranslate, so i checked the compatibility and the error was resolved (for me at least) with this change. maybe this will help someone if they also run into it,,

![image](https://github.com/user-attachments/assets/827b87e0-5fe6-401a-9623-eba75de0c4bd)

all loaded fine :) Thanks again !

**Comment by Celtmant** [comment]
But my json file won't load, I don't know why, everything seems to be installed
